src/index.js

```diff
@@ -281,6 +287,8 @@ function _renderToString(vnode, context, opts, inner, isSvgMode, selectValue) {
 				// <textarea value="a&b"> --> <textarea>a&amp;b</textarea>
 				propChildren = v;
 			} else if ((v || v === 0 || v === '') && typeof v !== 'function') {
+				name = getAttributeNameInHtmlCase(name);
```

Comment:
Likely faster if this is done on L310:

```diff
-				s += ` ${name}="${encodeEntities(v)}"`;
+				s += ` ${getAttributeNameInHtmlCase(name)}="${encodeEntities(v)}"`;
```

Reply:
Need the transformed name for the `if (v === true || v === '')` block